lib/ecto/enum.ex

```diff
@@ -208,6 +208,22 @@ defmodule Ecto.Enum do
   @impl true
   def embed_as(_, %{embed_as: embed_as}), do: embed_as
 
+  @impl true
+  def format(%{mappings: mappings}) do
+    valid_options = Keyword.keys(mappings)
+    # build a natural string description from possible values
+    # `:value1, :value2, or :value3`
+    [last_option | first_options] = Enum.reverse(valid_options)
+
+    comma_options =
+      first_options
+      |> Enum.reverse()
+      |> Enum.map(fn option -> ":" <> Atom.to_string(option) end)
+      |> Enum.join(", ")
+
+    "Ecto.Enum with values #{comma_options} or :#{last_option}"
```

Comment:
```suggestion
    "Ecto.Enum with values #{inspect Keyword.keys(mappings)}"
```
Not sure if this will be popular, but since it's an error message for developers maybe this is alright?

Reply:
```suggestion
    [last_option | first_options] = Enum.reverse(valid_options)

    comma_options =
      first_options
      |> Enum.reverse()
      |> Enum.map(fn option -> ":" <> Atom.to_string(option) end)
      |> Enum.join(", ")

    "#Ecto.Enum<values:  #{inspect Keyword.keys(mappings)}>"
```